lib/DDG/Spice/Rain.pm

```diff
@@ -0,0 +1,76 @@
+package DDG::Spice::Rain;
+
+use strict;
+use DDG::Spice;
+
+primary_example_queries "is it raining?";
+secondary_example_queries "is it raining in New York City?";
+description "Check if it's raining";
+name "Rain";
+source "Forecast.io";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Rain.pm";
+topics "everyday";
+category "facts";
+attribution web => [ 'https://bibhas.in', 'Bibhas Debnath' ],
+            github => [ 'https://github.com/iambibhas', 'Bibhas'],
+            twitter => ['http://twitter.com/bibhasdn', 'Bibhas D'];
+
+spice to => 'http://beta.json-generator.com/api/json/get/GO60Xk7?callback={{callback}}';
+
+triggers any => "rain", "raining";
+
+spice proxy_cache_valid => '5m';
+
+my %rain = map { $_ => undef } (
+    'is it going to rain',
+    'going to rain',
+    'going to rain today',
+    'is it raining',
+    'is it raining here',
+    'is it raining now',
+    'is it going to rain here',
+    'is it raining today',
+    'is it going to rain today',
+    'going to rain today',
+    'is it raining yet',
+    'make it rain',
```

Comment:
I don't think this trigger, or the next, make much sense. I'd expect "make it rain" to have some animation of sorts...